suite-common/bluetooth/src/bluetoothActions.ts

```diff
@@ -68,6 +69,13 @@ const scanStatusAction = createAction(
     ({ status }: { status: BluetoothScanStatus }) => ({ payload: { status } }),
 );
 
+const setAutoconnectPolicyAction = createAction(
```

Comment:
For consistency with `AutoConnectPolicy`:
```suggestion
const setAutoConnectPolicyAction = createAction(
```